osbuild/testutil/__init__.py

```diff
@@ -1,3 +1,8 @@
 """
 Test related utilities
 """
+import shutil
+
+
+def has_executable(executable: str) -> bool:
```

Comment:
I love this! Will need it in another branch of mine :)

Reply:
Full disclosure - I was battling with myself if I should ask for a trivial unit test for this or not. But decided against it as it's a oneliner and unlikely to ever get refactored or changed. But the impulse is there :)

Reply:
Haha yea, but we'd 'just' be testing if `shutil` works correctly I think.

Reply:
Yeah, the argument in favor would be that if we ever refactor this helper we could accidentally break things and not notice (or only indirectly) and that a good test is documentation but both are extremely weak in this case so I let it go :)